keywords.md

```diff
@@ -1,12 +1,17 @@
 ---
-title: 关键字和保留字
+title: 关键字
 category: reference
-aliases: ['/docs-cn/dev/reference/sql/language-structure/keywords-and-reserved-words/']
+summary: 本文介绍 TiDB 的关键字。
+aliases: ['/docs-cn/dev/keywords-and-reserved-words/']
```

Comment:
```suggestion
aliases: ['/docs-cn/dev/keywords-and-reserved-words/','/docs-cn/dev/reference/sql/language-structure/keywords-and-reserved-words/']
```

Reply:
@TomShawn Oops, should it be `/docs-cn/dev/keywords` instead of `/docs-cn/dev/keywords-and-reserved-words`?

Reply:
Aliases are always old paths. `/docs-cn/dev/keywords-and-reserved-words` is correct because this path is already there. We use an alias to make sure the old path is directed to `/docs-cn/dev/keywords`.